chain/chain/src/store.rs

```diff
@@ -1817,26 +1817,49 @@ impl<'a> ChainStoreUpdate<'a> {
             .insert((*hash, shard_id), outgoing_receipts);
     }
 
+    /// For all the outgoing receipts generated in block `hash` at the shards we are tracking
+    /// in this epoch,
+    /// save a mapping from receipt ids to the destination shard ids that the receipt will be sent
+    /// to in the next block.
+    /// Note that this function should be called after `save_block` is called on this block because
+    /// it requires that the block info is available in EpochManager.
```

Comment:
what happens if block_info is not in EpochManager?

Does this panic? or just return an empty list of receipts (and then the receipt_id_to_shard_id is not set)?

could we check this condition somehow programatically?